services/tenant-ui/frontend/src/components/reservation/ReservationConfirmation.vue

```diff
@@ -9,6 +9,9 @@
         {{ $t('reservations.submitted') }} <br />
         {{ $t('reservations.emailSentTo', [email]) }}
       </p>
+      <p class="text-center" v-if="pwd">
```

Comment:
Can hit an `npm run lintfix` in `frontend` to auto-fix these lint errors

Reply:
Thanks for the tip, run a lint and fixed them 🙂